code/modules/mob/emote.dm

```diff
@@ -45,20 +45,36 @@
 
 /datum/emote/help/run_emote(mob/user, params, type_override, intentional)
 	. = ..()
-	var/list/keys = list()
+	var/list/base_keys = list()
+	var/list/all_keys = list()
+	var/list/species_emotes = list()
 	var/list/message = list("Available emotes, you can use them with say \"*emote\": ")
```

Comment:
**[OPTIONAL]** This isn't technically related to the PR but I figured I'd suggest it anyway:

Bolding and adding a newline to these messages makes the emote list a lot easier to read, IMO:
**Before:**
![before](https://user-images.githubusercontent.com/57483089/168479470-2813ce7e-901b-4b22-8c79-809b6e691d9f.PNG)
**After:**
![after](https://user-images.githubusercontent.com/57483089/168479471-ba4d43b2-9a28-4d95-8b36-c76eb7f35fc0.PNG)
```suggestion
	var/list/message = list("<b>Available emotes, you can use them with say \"*emote\":</b> \n")
```